kotlin-js-store/yarn.lock

```diff
@@ -0,0 +1,3108 @@
+git add src comm
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
this file should not be committed, right?

Reply:
I've always committed the lock files that are generated from package managers in the past (package-lock.json on my past NPM projects) to ensure the dependencies are specific.

That being said - how did that stray `git` command get in there? 🤦   At a minimum I'll fix that, but definitely can remove this file if you think it's not entirely necessary

Reply:
- [ ] Check what other Square KMP projects do